<?xml version="1.0" encoding="UTF-8"?>

<launch>
    <rosparam command="load" file="$(find eagleye_rt)/config/eagleye_config.yaml"/>

     <group ns="eagleye">
        <node pkg="eagleye_rt" name="navpvt2rtk_node" type="navpvt2rtk" output="screen">
            <param name ="ublox_navpvt_topic" value="/ublox/navpvt"/>
            <param name ="nav_sat_fix_topic" value="/ublox/nav_sat_fix"/>
        </node>
    </group>

    <include file="$(find eagleye_rt)/launch/eagleye_rt.launch"/>
</launch>